README.md

```diff
@@ -19,22 +19,22 @@ More information about this product can be found [here](http://wso2.com/api-mana
 2. Start ActiveMQ using `` ./bin/activemq start ``
 
 ### Running WSO2 Identity Server
-1. Download the WSO2 Identity Server from [here](http://wso2.com/identity-and-access-management#download)
-2. Extract the downloaded wso2is-5.3.0.zip file
-3. Go to the wso2is-5.3.0/bin directory and execute the wso2server.sh script using the command ``./wso2server.sh start``
+1. Download the WSO2 Identity Server 5.4.0 Alpha10 or above from [here](http://wso2.com/identity-and-access-management#download)
+2. Extract the downloaded wso2is-5.4.0.zip file
+3. Go to the wso2is-5.4.0/bin directory and execute the wso2server.sh script using the command ``./wso2server.sh start``
 
 ### Running WSO2 API Manager Analytics
-1. Find the API Manager analytics distribution from product-apim/analyzer/target
+1. Find the API Manager analytics distribution download or from product-apim/analyzer/target if built from source
```

Comment:
Download the latest Analytics distribution from [Releases](https://github.com/wso2/product-apim/releases) OR if you built from source, find the binary from product-apim/analyzer/target.